docs/prebuilt.md

```diff
@@ -0,0 +1,67 @@
+# Prebuilt Binaries
+
+The prebuilt binaries for [rv32emu](https://github.com/sysprog21/rv32emu) are prepared primarily because the [RISC-V Sail Model](https://github.com/riscv/sail-riscv) executable is required for the [RISC-V Architecture Test](https://github.com/riscv-non-isa/riscv-arch-test), and selected RISC-V ELF files are useful for ISA simulation validation and testing.
+Some of these prebuilt binaries are stored in [rv32emu-prebuilt](https://github.com/sysprog21/rv32emu-prebuilt).
+During testing or benchmarking, these binaries are automatically downloaded into the `build/linux-x86-softfp/` and `build/riscv32/` directories by default.
+The RISC-V binaries are compiled using the [xPack RISC-V GCC toolchain](https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack) with the options `-march=rv32im -mabi=ilp32`.
+The x86 binaries are compiled by GCC with `-m32 -mno-sse -mno-sse2 -msoft-float` options and use [ieeelib](https://github.com/sysprog21/ieeelib) as the soft-fp library.
+
+To fetch the prebuilt binaries manually, run:
+
+```shell
+$ make artifact
+```
+
+Or build the binaries from scratch (the RISC-V cross-compiler is required):
+
+```shell
+$ make artifact ENABLE_PREBUILT=0 [CROSS_COMPILE=<COMPILER_PREFIX>]
+```
+
+The compiler prefix varies according to the used toolchain, such as `riscv-none-elf-`, `riscv32-unknown-elf-`, etc.
+
+The prebuilt binaries in `rv32emu-prebuilt` are built from the following repositories and resources:
+
+- [ansibench](https://github.com/sysprog21/ansibench)
+    - coremark
+    - stream
+    - nbench
+- [rv8-bench](https://github.com/sysprog21/rv8-bench)
+    - aes
+    - dhrystone
+    - miniz
+    - norx
+    - primes
+    - qsort
+    - sha512
+- `captcha` : See [tests/captcha.c](tests/captcha.c)
```

Comment:
> Add `/` to the beginning of the link because the document is not located in the root of the repository, which causes the link to be broken.

The single slash (`/`) seems not to redirect to the repository root. Or just remove the hyperlink for simplicity?

Reply:
> > Add `/` to the beginning of the link because the document is not located in the root of the repository, which causes the link to be broken.
> 
> The single slash (`/`) seems not to redirect to the repository root. Or just remove the hyperlink for simplicity?

Did you try `[tests/captcha.c](/tests/captcha.c)` ?

Reply:
> Did you try `[tests/captcha.c](/tests/captcha.c)` ?

So weird, it works. I remember it reported 404 a couple of months ago.

Reply:
> > Did you try `[tests/captcha.c](/tests/captcha.c)` ?
> 
> So weird, it works. I remember it reported 404 a couple of months ago.

Let's proceed with this.